fala/apps/adviser/views.py

```diff
@@ -248,6 +168,7 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
+        context["is_single_category_search"] = self.request.GET.get("is_single_category_search", False)
```

Comment:
Does this not make the URL `/search?is_single_category_search=True?`

Reply:
Yes, it'll be `/search?categories=hlpas&single_category_search_redirect=true&postcode=se1`